src/frontend/components/SubscribeScheduledVideo/index.spec.tsx

```diff
@@ -5,18 +5,24 @@ import {
   waitFor,
   screen,
 } from '@testing-library/react';
-import { Grommet } from 'grommet';
-import { videoMockFactory } from 'utils/tests/factories';
 import fetchMock from 'fetch-mock';
+import { Grommet } from 'grommet';
 import React from 'react';
+import { useVideo } from 'data/stores/useVideo';
+import { liveState } from 'types/tracks';
+import { Deferred } from 'utils/tests/Deferred';
+import { videoMockFactory } from 'utils/tests/factories';
 import { wrapInIntlProvider } from 'utils/tests/intl';
 import { SubscribeScheduledVideo } from '.';
 let mockDecodedJwt: any;
 jest.mock('jwt-decode', () => jest.fn());
 jest.mock('data/appData', () => ({
-  appData: { jwt: 'cool_token_m8' },
+  appData: { jwt: 'cool_token_m8', uploadPollInterval: 10 },
   getDecodedJwt: () => mockDecodedJwt,
 }));
+jest.mock('data/sideEffects/pollForLive', () => ({
+  pollForLive: jest.fn().mockResolvedValue(null),
+}));
```

Comment:
```suggestion
```

Reply:
If I don't add this mock I have a warning error